### DISCUSSION TITLE
How to add a column to table, column calculated from different data provider

### DISCUSSION THREAD

**Discussion by Loris-F** [Budibase/budibase#5657] [Help]
Hello, 
is it possibile to do create a calculated field to be show in a table component?
Try to explain the situation:
I have a big table show with "data provider" component and "Table" component.
I need to show in the grid of the table a calculated filed that is obtained with a different query from database.
For example in table1 with lot of records there are many shipments, one record per shipment.
Need to show the price that i in different table with relation one-o-one with first table.
How to obtain the new column?
thanks

**Comment by melohagan** [comment]
Hello @Loris-F 

Are you using a relational database? You could try using custom queries: https://docs.budibase.com/docs/custom-queries

For which you can provide a schema that will be the columns of your table.
